src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_OF_BOUND = 100;
```

Comment:
Please, read about `nextInt(int bound) `upper bounds. No, maximum possible value from random generating will be 99.
[Java doc](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#nextInt-int-)